### PR TITLE
Start using Phenix

### PR DESCRIPTION
**Pull request by pschambacher**
:octopus::computer:

/cc @bquorning @grosser 

### Steps to merge
* [ ] :+1: from the team

### Risks
* None

### CODE REVIEW

File: test/connection_switching_test.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
 require File.expand_path('../helper', __FILE__)
 
 describe "connection switching" do
-  schema_initialized = false
-
   before do
-    if !schema_initialized
-      init_schema
```
Comment (grosser) [inline test/connection_switching_test.rb]: so we don't need the schema anymore ?

File: test/schema_dumper_extension_test.rb [hunk 2]
```diff
@@ -15,14 +15,14 @@
 
       after do
         schema_file.unlink
-        init_schema
+        Phenix.burn!
```
Comment (grosser) [inline test/schema_dumper_extension_test.rb]: do we always need to tear down the db or just let rise handle it ?

Comment (pschambacher) [inline test/schema_dumper_extension_test.rb]: We don't necessarily need it but it keeps your local db cleaner to have it. Ideally we'd need some RSpec `after(:suite)` handler.

**Comment by bquorning** [comment]
Any change of getting this merged in?

**Comment by pschambacher** [comment]
I'd need to rebase and get approval for merge.

**Comment by pschambacher** [comment]
Ping @grosser @osheroff 
@bquorning I rebased and upgraded the ruby version to fix some random failure with Ruby 2.2.4 (as you can still see on Travis)

**Comment by grosser** [comment]
👍

**Comment by bquorning** [comment]
:+1:

**Pull request merged**